upi/metal/main.tf

```diff
@@ -45,6 +43,7 @@ resource "matchbox_profile" "master" {
   args = concat(
     local.kernel_args,
     ["coreos.inst.ignition_url=${var.matchbox_http_endpoint}/ignition?cluster_id=${var.cluster_id}&role=master"],
+    "${var.pxe_kernel_args}",
```

Comment:
I think after looking at https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-test/pr-logs/pull/openshift_installer/3865/pull-ci-openshift-installer-master-e2e-metal/1281237439557406720#1:build-log.txt%3A328, it looks like you'd need `s/"${var.pxe_kernel_args}"/"var.pxe_kernel_args`